src/IdentityServer4.WsFederation/ResponseProcessing/SignInResponseGenerator.cs

```diff
@@ -82,7 +82,16 @@ protected async Task<ClaimsIdentity> CreateSubjectAsync(SignInValidationResult r
                 Subject = result.User,
                 RequestedClaimTypes = requestedClaimTypes,
                 Client = result.Client,
-                Caller = "WS-Federation"
+                Caller = "WS-Federation",
+                // VIM-132 I'm not sure if this is correct, but this is required by our implementation of GetProfileDataAsync.
+                RequestedResources = new Resources
```

Comment:
could you please remove all the stuff that is not part of the concrete issue mentioned in the title of the PR. This makes it easier for me and others to follow what is really changes after this PR and minimizes the impact that something would break for other users.

Reply:
Hello Alexej, I'm very sorry.  I did not intend to commit all of this.  Some of it is just my customization that I intended to keep in my own fork.  I have never used a fork before.  I think these extra commits to my fork's master made it into this pull request.  

I will make some branches and fix all of this.  Very sorry to waste your time.

Reply:
Hi Jess, no worry I am happy you wanna contribute :)